### PR TITLE
Update/rest changes 2

### PR DESCRIPTION
**Pull request by JC000**
Changes based on feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,25 +69,24 @@ Example responses:
...
 ```
-The above response would indicate an existing object has been updated (existing object with the given UUID was found).  Note that a PUT operation on an existing record may also result in "403 Forbidden" error.
+The above response would indicate an existing object has been updated (existing object with the given ID was found).  Note that a PUT operation on an existing record may also result in "403 Forbidden" error.
 
 
 ### List - `store/{bucket}` endpoint
 This endpoint allows for `GET` to retrieve all the items in the bucket organized by the id.
 
-An example response will look like where "earth" is the id of the item. An optional header `X-Midt-Owner` can be sent with the request.  If supplied, only items with secrets matching the supplied value will be returned in the list. If items from all owners are desired, use the `X-Midt-Admin-Token` header. 
+An example response will look like where "earth" is the id of the item. An optional header `X-Midt-Owner` can be sent with the request.  If supplied, only items with secrets matching the supplied value will be returned in the list. Only authorized requests can retrieve items from all owners.
```
Comment (schmidtw) [inline README.md:83]: `earth` is an invalid id reference.  Change to `7e8c5f378b4addbaebc70897c4478cca06009e3e360208ebd073dbee4b3774e7`

File: README.md [hunk 2]
```diff
@@ -32,33 +32,31 @@ By participating, you agree to this Code.
...
 
-### Create Individual Item - `store/{bucket}/{uuid}` endpoint
+### Create Individual Item - `store/{bucket}/{id}` endpoint
 This endpoint allows for clients to `PUT` an object into argus.  The placeholder variables in the path must contain:
 
 * _bucket_ - The name used to indicate the resource type of which the stored data represents.  A plural form of a noun word should be used for stylistic reasons.
-* _uuid_ - The unique ID within the name space of the containing bucket.  It is recommended this value is the resulting value of a SHA256 calculation, using the unique attributes of the object being represented (e.g. `SHA256(<identifier>)`).  This will be used by argus to determine uniqueness of objects being stored or updated.
+* _id_ - The unique ID within the name space of the containing bucket.  It is recommended this value is the resulting value of a SHA256 calculation, using the unique attributes of the object being represented (e.g. `SHA256(<common_name>)`).  This will be used by argus to determine uniqueness of objects being stored or updated.  argus will not accept any values for this attribute that is not a 64 character hex string.
```
Comment (schmidtw) [inline README.md:39]: I think we should accept a wider range of characters. `[a-zA-Z0-9_=-]` to allow for base64url encoded values or ulid encoded values.  Restricting the special characters helps eliminate the potential for injection issues.

**Review by schmidtw — CHANGES_REQUESTED** [review]

**Review by joe94 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (JC000) [reply README.md:39]: If you use base64url encoding, then you're pretty much guaranteed that it won't be 64 characters long.  The idea is that we want clients to use SHA256.  If a client wants to deviate from that, then they'll need to make their own adjustments for something like base64url encoding.

File: README.md (hunk 1 above)
Comment (JC000) [reply README.md]: Updated.

**Review by schmidtw — APPROVED** [review]

**Pull request merged**